index.html

```diff
@@ -0,0 +1,2620 @@
+                      </p>
+                      <p class="text-secondary">
+                        BY FRANK CILLUFFO AND STEWART BAKER
+                      </p>
+                    </div>
+                  </div>
+
+                  <div class="media border-bottom">
+                    <img
+                      src="img/frank-cilluffo-stewart-baker.webp"
+                      class="align-self-center mr-3"
+                      alt="twoguys"
+                    />
+                    <div class="media-body">
+                      <p class="pt-1 mb-1 font-weight-bold">
+                        U.S. Supply Chain: Stop Driving Blind
+                      </p>
+                      <p class="text-secondary">
+                        BY FRANK CILLUFFO AND STEWART BAKER
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              
+
+              <div class="col-12 p-2 d-none d-md-block">
+                <div class="row">
+                  <!--    SPONSORED INSIGHT -->
+                  <div class="pt-3">
+                    <h6 class="text-primary font-weight-bold">
+                      SPONSORED INSIGHT
+                    </h6>
+                    <div class="card border-box p-4">
+                      <div class="article-body">
+                        <h5 class="card-title font-weight-bold">
+                          Hungary: From Manufacturing Hub to Innovation and R&D
+                        </h5>
+                        <p class="card-text text-secondary">
+                          With its pro-business agenda and open-border policy
+                          for investors, the Central European nation is hoping
+                          to bounce back from the COVID-19 crisis, while making
+                          greater strides toward innovation, research and
+                          development.
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- FREE SIGN UP -->
+                  <div class="newsletter-signup mt-5">
+                    <form class="free-signup">
+                      <label
+                        for="email_address"
+                        class="d-flex mt-2 align-items-center"
+                      >
+                        <span class="icon-signup"> </span>
+                        <span class="free-signup-text mb-1"
+                          >Start your day with our top 5 articles</span
+                        >
+                      </label>
+                      <input
+                        type="text"
+                        id="email_address1"
+                        class="form-control"
+                        placeholder="Email address"
+                      />
+                      <input
+                        type="submit"
+                        class="btn btn-danger mt-2 btn-block"
```

Comment:
- [ ] I see that you are using the `text-danger` and `button-danger` sometimes to get a red text and background in your clone, while using Bootstrap classes it's great it can be a problem in this case.
The shade of red the NewsWeek site uses is not the same red the Bootstrap classes have.

Please go through your clone and make sure that all the places where the shade of red differ from the original site are fixed.

Reply:
done